<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonsSettings</class>
 <widget class="QDialog" name="ButtonsSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ButtonsSettings</string>
  </property>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okayButton">
      <property name="text">
       <string>Okay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>10</y>
     <width>46</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="buttonY">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GUI/Resources/Xbox_Y_button.png) 0 0 0 0;
border-top: 3px transparent;
border-bottom: 3px transparent; 
border-right: 10px transparent;
border-left: 10px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonB">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GUI/Resources/Xbox_B_button.png) 0 0 0 0;
border-top: 3px transparent;
border-bottom: 3px transparent; 
border-right: 10px transparent;
border-left: 10px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonX">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GUI/Resources/Xbox_X_button.png) 0 0 0 0;
border-top: 3px transparent;
border-bottom: 3px transparent; 
border-right: 10px transparent;
border-left: 10px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonA">
      <property name="styleSheet">
       <string notr="true">border-image: url(:/GUI/Resources/Xbox_A_button.png) 0 0 0 0;
border-top: 3px transparent;
border-bottom: 3px transparent; 
border-right: 10px transparent;
border-left: 10px transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select Action&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="cursorWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="Y_Combo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LeftClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RightClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Start(W.I.P)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollUp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollDown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OpenVirtualKeyboard</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="B_Combo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LeftClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RightClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Start(W.I.P)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollUp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollDown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OpenVirtualKeyboard</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select Action&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="cursorWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="X_Combo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LeftClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RightClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Start(W.I.P)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollUp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollDown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OpenVirtualKeyboard</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select Action&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="cursorWidth">
    <number>1</number>
   </property>
  </widget>
  <widget class="QComboBox" name="A_Combo">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>128</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LeftClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RightClick</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Start(W.I.P)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollUp</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ScrollDown</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OpenVirtualKeyboard</string>
    </property>
   </item>
  </widget>
  <widget class="QTextEdit" name="textEdit_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Select Action&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="tabStopWidth">
    <number>80</number>
   </property>
   <property name="cursorWidth">
    <number>1</number>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>textEdit</zorder>
  <zorder>Y_Combo</zorder>
  <zorder>B_Combo</zorder>
  <zorder>textEdit_2</zorder>
  <zorder>X_Combo</zorder>
  <zorder>textEdit_3</zorder>
  <zorder>A_Combo</zorder>
  <zorder>textEdit_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections/>
</ui>
